design/sketch.webidl

```diff
@@ -437,15 +437,32 @@ enum WebGPUStoreOp {
     "store",
 }
 
-dictionary WebGPURenderPassAttachmentDescriptor {
+dictionary WebGPURenderPassColorAttachmentDescriptor {
     WebGPUTextureView attachment;
+
     WebGPULoadOp loadOp;
     WebGPUStoreOp storeOp;
+    float clearR;
+    float clearG;
+    float clearB;
+    float clearA;
+};
+
+dictionary WebGPURenderPassDepthStencilAttachmentDescriptor {
+    WebGPUTextureView attachment;
+
+    WebGPULoadOp depthLoadOp;
+    WebGPUStoreOp depthStoreOp;
+    float clearDepth;
+
+    WebGPULoadOp stencilLoadOp;
+    WebGPUStoreOp stencilStoreOp;
+    uint32_t clearStencil;
 };
 
 dictionary WebGPURenderPassDescriptor {
-    sequence<WebGPURenderPassAttachmentDescriptor> colorAttachments;
-    WebGPURenderPassAttachmentDescriptor depthStencilAttachmaent;
+    sequence<WebGPURenderPassColorAttachmentDescriptor> colors;
```

Comment:
Let's continue calling them colorAttachments

Reply:
How about depthStencil? I feel `colorAttachments` is a bit long but didn't come up with a better alternative than `colors`.

Reply:
I think `colorAttachments` and `depthStencilAttachment` are the best options right now even though they're rather long.